meeting-notes/2018-07-12--protocol-design-relay.md

```diff
@@ -0,0 +1,7 @@
+* Choosing relay
+    * use xor metric?
+
+* With autonat telling that we are reachable and not being able to reach a peer directly, we could tell the other peer our public addrs with a relay connect message
+    * should the address list be signed/authorized in some way?
+* Extend indentify protocol to have protocol-specific metadata
+* Don't act as a relay relay when autonat says we won't help
```

Comment:
This should go into https://github.com/libp2p/developer-meetings/tree/master/notes